### PR TITLE
Example improvement

### PR DESCRIPTION
**Pull request by EmilianoSanchez**
-Dependencies and README.md were updated
-Dependencies, such as 'events' and 'utils', were removed since they are not required for the last @splitsoftware/splitio package

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,22 +68,14 @@ Runs your app in development mode.
...
-
 #### `npm test`
 
 Runs the [jest](https://github.com/facebook/jest) test runner on your tests.
 
 Note: No test cases have been added since this is an example app.
 If you're looking for how to test with Split SDK, you can:
    * mock the module import, see Jest documentation for that [here](https://facebook.github.io/jest/docs/en/jest-object.html#jestmockmodulename-factory-options)
-   * use the localhost (offline) mode of the JavaScript Split SDK, more information [here](https://docs.split.io/docs/javascript-sdk-overview#section-localhost-mode)
+   * use the localhost (offline) mode of the JavaScript Split SDK, more information [here](https://docs.split.io/docs/javascript-sdk-overview#localhost-mode)
```
Comment (NicoZelaya) [inline README.md]: We should use the new URL, pointing to help.split.io

File: README.md [hunk 2]
```diff
@@ -82,56 +100,29 @@ Like `npm start`, but also attempts to open your app on a connected Android devi
...
-3. If that doesn't work, open the Simulator, and under the app menu select `Reset Contents and Settings...`. After that has finished, quit the Simulator, and re-run `npm/yarn run ios`.
-
-### QR Code does not scan
-
-If you're not able to scan the QR code, make sure your phone's camera is focusing correctly, and also make sure that the contrast on the two colors in your terminal is high enough. For example, WebStorm's default themes may [not have enough contrast](https://github.com/react-community/create-react-native-app/issues/49) for terminal QR codes to be scannable with the system barcode scanners that the Expo app uses.
-
-If this causes problems for you, you may want to try changing your terminal's color theme to have more contrast, or running Create React Native App from a different terminal. You can also manually enter the URL printed by the packager script in the Expo app's search bar to load it manually.
+## About Split
```
Comment (NicoZelaya) [inline README.md]: Same here. You can take this section from the Confluence page (Readme structure)

File: package.json [hunk 3]
```diff
@@ -20,16 +20,9 @@
...
-    "events": "^3.0.0",
-    "expo": "^33.0.7",
-    "extend": "^3.0.2",
-    "merge": "^1.2.1",
-    "randomatic": "^3.1.1",
-    "react": "16.8.3",
-    "react-native": "0.59.8",
-    "rox-react-native": "^2.3.7",
-    "util": "^0.11.1"
+    "@splitsoftware/splitio": "^10.8.4-canary.2",
```
Comment (NicoZelaya) [inline package.json]: Let's remove the canary here and use the 10.8.4 stable (don't forget to update the package-lock.json along with it)

File: package.json [hunk 4]
```diff
@@ -20,16 +20,9 @@
...
+    "expo": "^34.0.1",
+    "react": "16.8.6",
+    "react-native": "https://github.com/expo/react-native/archive/sdk-34.0.0.tar.gz"
```
Comment (NicoZelaya) [inline package.json]: Any particular reason to use the link to the .tar.gz instead of downloading from npm repository?

Comment (EmilianoSanchez) [reply package.json]: This dependency was added by "expo init" command. Changing it to one using semantic versioning spec, generated a Warning in the expo UI.

**Pull request merged**